README.md

```diff
@@ -40,7 +41,7 @@ Instructions for using Mythril Classic are found on the [Wiki](https://github.co
 For support or general discussions please join the Mythril community on [Discord](https://discord.gg/E3YrVtG).
 
 ## Bulding the Documentation
-Mythril Classic's documentation is contained in the `docs` folder. It is based on Sphinx and can be built using the Makefile contained in the subdirectory:
+Mythril Classic's documentation is contained in the `docs` folder and published to [Read the Docs](https://mythril-classic.readthedocs.io/en/master/). It is based on Sphinx and can be built using the Makefile contained in the subdirectory:
```

Comment:
shouldn't it be `and is published to` ?I am not an english grammer guy so not totally sure

Reply:
Hmm... I think they both work, at least they both sound fine to me.

An object is something and is something else.
An object is something and something else.

Adding an is might read a bit nicer though, so I'll go ahead and add it.